webapp/javascript/services/users.ts

```diff
@@ -81,6 +81,45 @@ export async function loadCurrentUser(): Promise<
   return Result.err<User, RequestError>(response.error);
 }
 
+export async function logIn({
+  username,
+  password,
+}: {
+  username: string;
+  password: string;
+}): Promise<any> {
```

Comment:
Why any? 

If we do know about the response type, then let's actually parse it to validate it's correct.

If we don't know/don't care, then let's just type it as `unknown`.